src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -30,6 +30,26 @@ public bool IsVerboseEnabled
             }
         }
 
+        // Verbosity is Error - so it is always sent to portal; Keyword is Diagnostics so throttling is not applied.
+        [Event(1,
+            Message = "It is a diagnostics message. Backoff logic was disabled. Transmission will be resolved.",
```

Comment:
nitpick: If this is a message that will be in the logs for us to trouble shoot with I might make it more concise: "Diagnostic message: backoff logic disabled, transmission will be resolved."...
Also, what is "resolved" meant to imply, is transmission is currently not resolving to an endpoint?